### PR TITLE
Pullbranch

### PR DESCRIPTION
**Pull request by JanetMo**
Hello,
I tried to add several resources from my personal web3-resources list. 
Hope there is something helpful amongst them. (-:

### CODE REVIEW

File: content/guides/defi-developer-roadmap.md
Comment (wslyvh) [inline content/guides/defi-developer-roadmap.md:1]: duplicate

File: content/earn/pointer.md
Comment (wslyvh) [inline content/earn/pointer.md:1]: no longer active

File: content/courses/devpill.md
Comment (wslyvh) [inline content/courses/devpill.md:1]: duplicate. Listed under websites

File: content/websites/kernel-resources.md
Comment (wslyvh) [inline content/websites/kernel-resources.md:1]: You added Kernel community above. I'd prefer that one and consider this as a duplicate.

**Review by wslyvh — CHANGES_REQUESTED** [review]
Awesome! This is a great list! Thanks for taking the time to submit all of these. 

Left a few comments, but happy to merge/add most of these

**Comment by JanetMo** [comment]
Sure, feel free to add what fits. 
The page is a great initiative!

**Comment by JanetMo** [comment]
Or should I delete the commented ones? 
It's my first time doing a pull request 🙈 (-:

**Comment by wslyvh** [comment]
> Or should I delete the commented ones? It's my first time doing a pull request see_no_evil (-:

That's great. Always nice to see first-time contributions :) 

If you feel like it, please go ahead an update the PR. 
In this case it's not too much work, but it's more common to have a PR ready to merge without requiring changes. 
But happy to help if you need any

**Comment by JanetMo** [comment]
Thanks, hope it's fine now (-:

**Comment by wslyvh** [comment]
Amazing.. Merged! Thanks again for contributing :)

**Comment by JanetMo** [comment]
You`re welcome! I'll check once in a time if I find something to add. 

Look forward to the development of the page.  (-:

**Pull request merged**